test/new_relic/control/security_interface_test.rb

```diff
@@ -23,11 +31,15 @@ def test_initialization_short_circuits_when_the_security_agent_is_disabled
       end
 
       refute_predicate NewRelic::Control::SecurityInterface.instance, :agent_started?
+      assert_metrics_recorded 'Supportability/Ruby/SecurityAgent/Agent/Enabled/disabled'
+      assert_metrics_recorded 'Supportability/Ruby/SecurityAgent/Enabled/enabled'
     end
     logger.verify
   end
 
   def test_initialization_short_circuits_when_the_security_is_disabled
+    reset_supportability_metrics
```

Comment:
This is a nice helper method that will perform a commonly needed operation for multiple tests.

Perhaps a similar helper can be used for the 2 `assert_metrics_recorded` calls that are needed by so many tests.

Reply:
Great idea! There are a few places where both are enabled, and I've cleaned that up into a method https://github.com/newrelic/newrelic-ruby-agent/pull/2286/commits/da5b18cabce2aa041e459e8bb7d4f7322437ef94